taipy/core/setup.py

```diff
@@ -45,6 +45,9 @@
     "mssql": ["pyodbc>=4,<4.1"],
     "mysql": ["pymysql>1,<1.1"],
     "postgresql": ["psycopg2>2.9,<2.10"],
+    "parquet": ["fastparquet==2022.11.0"],
```

Comment:
This is duplicated. In ParquetDataNode, we actually allow 2 engines, default is `pyarrow`, and the optional one is `fastparquet`.

So if we want to make parquet as optional, the `pyarrow` should also be in here

Reply:
Please do not set the version of dependencies with `==`.